### PR TITLE
Add speedo log file performance metrics

### PR DESCRIPTION
**Pull request by paweldudzinski**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/performance/one-page.md [hunk 1]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
 
+Detailed metric values can be found in performance.json.
```
Comment (lysannep) [inline docs/performance/one-page.md:312]:
```suggestion
Detailed metric values can be found in the `performance.json` log file. 
```

Comment (lysannep) [inline docs/performance/one-page.md:312]:
```suggestion
Detailed metric values can be found in the `performance.json` log file. 
```

File: docs/performance/one-page.md [hunk 2]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
```
Comment (lysannep) [inline docs/performance/one-page.md:311]: We should add a heading for this table. 
```suggestion 

#### Detailed Metric Values
```

Comment (lysannep) [inline docs/performance/one-page.md:311]: We should add a heading for this table. 
```suggestion 

#### Detailed Metric Values
```

File: docs/performance/one-page.md [hunk 3]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
 
+Detailed metric values can be found in performance.json.
+This file is generated when speedo is run with `-l` parameter.<br />
+The log file consists following metrics:
```
Comment (lysannep) [inline docs/performance/one-page.md:314]:
```suggestion
The log file consists of the following metrics:
```

Comment (lysannep) [inline docs/performance/one-page.md:314]:
```suggestion
The log file consists of the following metrics:
```

File: docs/performance/one-page.md [hunk 4]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+
+|Metric|Description|
+|---|------|
+`rrt`|Specifies the time it took for any redirects to happen.
```
Comment (lysannep) [inline docs/performance/one-page.md:318]:
```suggestion
`rrt`|The time it takes for any redirects to happen.
```

Comment (lysannep) [inline docs/performance/one-page.md:318]:
```suggestion
`rrt`|The time it takes for any redirects to happen.
```

File: docs/performance/one-page.md [hunk 5]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
 
+Detailed metric values can be found in performance.json.
+This file is generated when speedo is run with `-l` parameter.<br />
```
Comment (lysannep) [inline docs/performance/one-page.md:313]: Does the `-l` parameter also work with the`analyze` command?

Comment (lysannep) [inline docs/performance/one-page.md:313]: Does the `-l` parameter also work with the`analyze` command?

File: docs/performance/one-page.md [hunk 6]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+The log file consists following metrics:
+
+|Metric|Description|
+|---|------|
+`rrt`|Specifies the time it took for any redirects to happen.
+`load`|The amount of time it takes for all page objects and dependent resources to be loaded.
+`score`|Overall Lighthouse performance score.
+`maxRtt`|Specifies max time it took for any redirects to happen.
```
Comment (lysannep) [inline docs/performance/one-page.md:321]:
```suggestion
`maxRtt`|The maximum amount of time it takes for any redirects to happen.
```

Comment (lysannep) [inline docs/performance/one-page.md:321]:
```suggestion
`maxRtt`|The maximum amount of time it takes for any redirects to happen.
```

File: docs/performance/one-page.md [hunk 7]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`numFonts` / `fontCount`|Number of fonts for webpage to render.
```
Comment (lysannep) [inline docs/performance/one-page.md:322]:
```suggestion
`numFonts` / `fontCount`|The number of fonts for the webpage to render.
```

Comment (lysannep) [inline docs/performance/one-page.md:322]:
```suggestion
`numFonts` / `fontCount`|The number of fonts for the webpage to render.
```

File: docs/performance/one-page.md [hunk 8]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
 
+Detailed metric values can be found in performance.json.
+This file is generated when speedo is run with `-l` parameter.<br />
+The log file consists following metrics:
+
+|Metric|Description|
```
Comment (lysannep) [inline docs/performance/one-page.md:316]: The [Metric Values](https://docs.saucelabs.com/performance/one-page/#metric-values) table includes a "Unit" column -- if applicable, we could add the same column to be consistent.

Comment (lysannep) [inline docs/performance/one-page.md:316]: The [Metric Values](https://docs.saucelabs.com/performance/one-page/#metric-values) table includes a "Unit" column -- if applicable, we could add the same column to be consistent.

File: docs/performance/one-page.md [hunk 9]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+|---|------|
+`rrt`|Specifies the time it took for any redirects to happen.
+`load`|The amount of time it takes for all page objects and dependent resources to be loaded.
+`score`|Overall Lighthouse performance score.
```
Comment (lysannep) [inline docs/performance/one-page.md:320]:
```suggestion
`score`|The overall Lighthouse performance score.
```

Comment (lysannep) [inline docs/performance/one-page.md:320]:
```suggestion
`score`|The overall Lighthouse performance score.
```

File: docs/performance/one-page.md [hunk 10]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`numTasks`|Total number of tasks run by a browser during a request.
+`numTasksOverXms`|Total number of tasks run by a browser during a request that took more that X milliseconds.
+`fontSize`|Size of the rendered fonts.
+`firstPaint`|The time it takes to render the first pixel on the page once the URL has been called.
+`imageSize`|Size of all loaded images.
+`numScripts`|Number of external scripts found while rendering page.
+`speedIndex`|The average time is takes the contents of a page to fully render.
+`throughput`|Network throughput measured during the test run.
+`imageCount`|Number of rendered images.
+`numRequests` / `requestsCount`|Number of requests to server.
```
Comment (lysannep) [inline docs/performance/one-page.md:332]:
```suggestion
`numRequests` / `requestsCount`|Number of requests to the server.
```

File: docs/performance/one-page.md [hunk 11]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`requestsSize`|Overall request size.
+`documentSize` / `totalByteWeight`|Overall document size.
+`totalTaskTime`|Time to fulfill all the scheduled tasks.
```
Comment (lysannep) [inline docs/performance/one-page.md:335]:
```suggestion
`totalTaskTime`|The amount of time it takes to fulfill all the scheduled tasks.
```

Comment (lysannep) [inline docs/performance/one-page.md:335]:
```suggestion
`totalTaskTime`|The amount of time it takes to fulfill all the scheduled tasks.
```

File: docs/performance/one-page.md [hunk 12]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`firstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.
+`lastVisualChange`|The amount of time it takes for the final visual element to display.
+`maxServerLatency`|Maximum amount of time when server staled.
+`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to firstPaint that is browser agnostic.
+`totalBlockingTime`|The amount of time that elapses between firstContentfulPaint and timeToFirstInteractive, which is a key indicator of lag.
+`serverResponseTime`|Formerly timeToFirstByte, this is the amount of time it takes to receive the first data response from the server.
+`firstContentfulPaint`|The time from when the page starts loading to when any part of the page's content is rendered on the screen. In this context, `content` can be text, images, elements, or canvas (non-white) elements. This does not mean that the page is fully rendered.
+`firstMeaningfulPaint`|The amount of time it takes for a page's primary body of content to render. This metric is replaced by largestContentfulPaint.
```
Comment (lysannep) [inline docs/performance/one-page.md:345]:
```suggestion
`firstMeaningfulPaint`|<span className="sauceRed">DEPRECATED</span> The amount of time it takes for a page's primary body of content to render. This metric is replaced by `largestContentfulPaint`.
```

Comment (lysannep) [inline docs/performance/one-page.md:345]:
```suggestion
`firstMeaningfulPaint`|The amount of time it takes for a page's primary body of content to render. This metric is replaced by `largestContentfulPaint`.
```

File: docs/performance/one-page.md [hunk 13]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`requestsSize`|Overall request size.
+`documentSize` / `totalByteWeight`|Overall document size.
+`totalTaskTime`|Time to fulfill all the scheduled tasks.
+`numStylesheets`|Number of stylesheets rendered.
+`domContentLoaded`|The point at which visual content is fully rendered and backend scripts begin to execute.
+`firstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.
+`lastVisualChange`|The amount of time it takes for the final visual element to display.
+`maxServerLatency`|Maximum amount of time when server staled.
+`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to firstPaint that is browser agnostic.
+`totalBlockingTime`|The amount of time that elapses between firstContentfulPaint and timeToFirstInteractive, which is a key indicator of lag.
```
Comment (lysannep) [inline docs/performance/one-page.md:342]:
```suggestion
`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.
```

Comment (lysannep) [inline docs/performance/one-page.md:342]:
```suggestion
`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.
```

File: docs/performance/one-page.md [hunk 14]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+
+|Metric|Description|
+|---|------|
+`rrt`|Specifies the time it took for any redirects to happen.
+`load`|The amount of time it takes for all page objects and dependent resources to be loaded.
+`score`|Overall Lighthouse performance score.
+`maxRtt`|Specifies max time it took for any redirects to happen.
+`numFonts` / `fontCount`|Number of fonts for webpage to render.
+`numTasks`|Total number of tasks run by a browser during a request.
+`numTasksOverXms`|Total number of tasks run by a browser during a request that took more that X milliseconds.
```
Comment (lysannep) [inline docs/performance/one-page.md:324]:
```suggestion
`numTasksOverXms`|Total number of tasks run by a browser during a request that took more than X milliseconds.
```

Comment (lysannep) [inline docs/performance/one-page.md:324]:
```suggestion
`numTasksOverXms`|Total number of tasks run by a browser during a request that took more than X milliseconds.
```

File: docs/performance/one-page.md [hunk 15]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`documentSize` / `totalByteWeight`|Overall document size.
+`totalTaskTime`|Time to fulfill all the scheduled tasks.
+`numStylesheets`|Number of stylesheets rendered.
+`domContentLoaded`|The point at which visual content is fully rendered and backend scripts begin to execute.
+`firstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.
+`lastVisualChange`|The amount of time it takes for the final visual element to display.
+`maxServerLatency`|Maximum amount of time when server staled.
+`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to firstPaint that is browser agnostic.
+`totalBlockingTime`|The amount of time that elapses between firstContentfulPaint and timeToFirstInteractive, which is a key indicator of lag.
+`serverResponseTime`|Formerly timeToFirstByte, this is the amount of time it takes to receive the first data response from the server.
```
Comment (lysannep) [inline docs/performance/one-page.md:343]:
```suggestion
`serverResponseTime`|Formerly `timeToFirstByte`, this is the amount of time it takes to receive the first data response from the server.
```

Comment (lysannep) [inline docs/performance/one-page.md:343]:
```suggestion
`serverResponseTime`|Formerly `timeToFirstByte`, this is the amount of time it takes to receive the first data response from the server.
```

File: docs/performance/one-page.md [hunk 16]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`numRequests` / `requestsCount`|Number of requests to server.
+`requestsSize`|Overall request size.
+`documentSize` / `totalByteWeight`|Overall document size.
+`totalTaskTime`|Time to fulfill all the scheduled tasks.
+`numStylesheets`|Number of stylesheets rendered.
+`domContentLoaded`|The point at which visual content is fully rendered and backend scripts begin to execute.
+`firstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.
+`lastVisualChange`|The amount of time it takes for the final visual element to display.
+`maxServerLatency`|Maximum amount of time when server staled.
+`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to firstPaint that is browser agnostic.
```
Comment (lysannep) [inline docs/performance/one-page.md:341]:
```suggestion
`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to `firstPaint` that is browser agnostic.
```

Comment (lysannep) [inline docs/performance/one-page.md:341]:
```suggestion
`firstVisualChange`|The time it takes for anything to be visually painted in the viewport. Calculated by video analysis, this is an alternative metric to `firstPaint` that is browser agnostic.
```

File: docs/performance/one-page.md [hunk 17]
```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
...
+`imageCount`|Number of rendered images.
+`numRequests` / `requestsCount`|Number of requests to server.
+`requestsSize`|Overall request size.
+`documentSize` / `totalByteWeight`|Overall document size.
+`totalTaskTime`|Time to fulfill all the scheduled tasks.
+`numStylesheets`|Number of stylesheets rendered.
+`domContentLoaded`|The point at which visual content is fully rendered and backend scripts begin to execute.
+`firstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.
+`lastVisualChange`|The amount of time it takes for the final visual element to display.
+`maxServerLatency`|Maximum amount of time when server staled.
```
Comment (lysannep) [inline docs/performance/one-page.md:340]:
```suggestion
`maxServerLatency`|The maximum amount of time it takes for the server to respond.
```

Comment (lysannep) [inline docs/performance/one-page.md:340]:
```suggestion
`maxServerLatency`|The maximum amount of time it takes for the server to respond.
```

**Review by lysannep — CHANGES_REQUESTED** [review]

File: docs/performance/one-page.md (hunk 3 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 4 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 1 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 2 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:311]: Done.

File: docs/performance/one-page.md (hunk 3 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 6 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 7 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 9 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 11 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 12 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done.

File: docs/performance/one-page.md (hunk 15 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:345]: Done

File: docs/performance/one-page.md (hunk 16 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:343]: Done

File: docs/performance/one-page.md (hunk 17 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:342]: Done

File: docs/performance/one-page.md (hunk 5 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:315]: @lysannep yes, the `-l` parameter works with `analyze` command

File: docs/performance/one-page.md (hunk 16 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:343]: Done

File: docs/performance/one-page.md (hunk 8 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done

File: docs/performance/one-page.md (hunk 13 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done

Comment (paweldudzinski) [reply docs/performance/one-page.md]: Done

File: docs/performance/one-page.md (hunk 14 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:326]: Done

File: docs/performance/one-page.md [hunk 18]
```diff
@@ -309,43 +309,45 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
 
-Detailed metric values can be found in performance.json.
+#### Detailed Metric Values
+
+Detailed metric values can be found in the `performance.json` log file.
 This file is generated when speedo is run with `-l` parameter.<br />
```
Comment (lysannep) [inline docs/performance/one-page.md:315]:
```suggestion
This file is generated with the `-m` parameter of either the `run` or `analyze` command.

```

**Review by lysannep — APPROVED** [review]
Looks good! ✨ I added one more suggestion.

File: docs/performance/one-page.md (hunk 18 above)
Comment (paweldudzinski) [reply docs/performance/one-page.md:315]: Done.

**Pull request merged**